modules/pel/peltool/peltool.py

```diff
@@ -509,6 +511,10 @@ def extractAndSummarizePEL(file: str, config: Config):
         try:
             eid, summary = parsePELSummary(stream, config)
             if eid :
+                if config.plid and summary['PLID'] != config.plid:
```

Comment:
Why are you preferring to check this here? You could add these checks in the parsePelFromPLIdAndSRCId function itself, since you already created a separate API for PLID and SRCID-based summaries and are only checking the value from the summaries key, correct?